require.context replacement doesn't work for nested paths on Windows

This plugin will rewrite the following
```js
const customTabsContext = require.context('@/components/tabs', true, /^.*(.vue)$/)
```
to something like this when using Windows:
```js
const customTabsContext = 
  (function() {
    var map = {
      './category1\Tab1.vue': import_context_module_0_0,
'./category1\Tab2.vue': import_context_module_0_1,
'./category2\Tab3.vue': import_context_module_0_2,

    };
    var req = function req(key) {
      return map[key] || (function() { throw new Error("Cannot find module '" + key + "'.") }());
    }
    req.keys = function() {
      return Object.keys(map);
    }
    return req;
  })()
```
Note that the keys in the map use backslashes `\` for nested paths. However to put a backslash into a string literal you need to escape it `\\`. Therefore `require.context` won't work for nested paths on Windows.

This works fine on Mac OS, as the keys in the map will use forward slashes `/`.

--- a/src/lib/resolve.ts
+++ b/src/lib/resolve.ts
@@ -1,5 +1,6 @@
 import fs from 'fs'
 import path from 'path'
+import { normalizePath } from 'vite'
 
 let uid = 0
 
@@ -96,7 +97,7 @@ export function resolveDir(dirname = './', deep = false, regexp = /^\.\//) {
     files = [files]
   }
 
-  files = files.map((file: string) => `./${path.relative(dirname, file)}`)
+  files = files.map((file: string) => normalizePath(`./${path.relative(dirname, file)}`))
   return files.filter((file: string) => {
     const moduleAbsolutePath = getModuleAbsolutePath(dirname, file)
     const isFile = !fs.statSync(moduleAbsolutePath).isDirectory()

